test/e2e/conf.js

```diff
@@ -38,14 +41,36 @@ if (process.env.TRAVIS) {
   }
 
   if (process.env.SAUCE_PLATFORM_NAME === 'iOS') {
+    config.port = 4723;
     var appiumiOS = require('./appium/ios-conf.js');
     config.multiCapabilities = appiumiOS.iosCapabilities;
   }
 
   else if (process.env.SAUCE_PLATFORM_NAME === 'android') {
+    config.port = 4723;
     var appiumAndroid = require('./appium/android-conf.js');
     config.multiCapabilities = appiumAndroid.androidCapabilities;
   }
+
+  else if (process.env.SAUCE_PLATFORM_NAME === 'windows') {
+    config.multiCapabilities =
+    [{
+      'browserName': 'internet explorer',
+      'browserVersion': 'latest',
+      'platformName': 'Windows 10',
+      'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
+      'build': process.env.TRAVIS_BUILD_NUMBER
+    },
+    {
+      'browserName': 'MicrosoftEdge',
+      'browserVersion': 'latest',
+      'platformName': 'Windows 10',
+      'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
+      'build': process.env.TRAVIS_BUILD_NUMBER
+    }];
+    config.exclude = ['specs/angular_spec.js', 'specs/dev_spec.js', 'specs/npm_spec.js',
```

Comment:
why exclude those tests?

Reply:
Some of the tests simply don't pass on IE 11. For example, getting the browser logs throws an error in IE, probably because it's not supported? I have to still dig deep.
```
dev_spec.js - expectToFindLogMessage
1) Dev Mode flows log a console message when tokens are not parsed from the URL after the Widget is rendered
Error: Failed expectation
        at /Users/vijetm/okta/okta-signin-widget/target/e2e/specs/dev_spec.js:116:21
```
I thought it's better to have some coverage first (that gives us confidence that the widget loads on IE) and then figure out why all these tests are failing on IE.
What do you suggest?

Reply:
sounds good to me. better to include all that are able to pass today?

Reply:
I can do that, but in most of the specs, out of 5 of them, 1 or 2 pass. 
That'll add a lot of `if browser == ie` statements. I don't think that's clean. This one spec has the most tests passing. So, I'll add this for now. Will create a JIRA to track other failures.